src/main/kotlin/com/software/modsen/driverservice/service/DriverService.kt

```diff
@@ -23,57 +25,61 @@ class DriverService(
     private val carRepository: CarRepository,
     private val driverProducer: DriverProducer
 ) {
-    fun getDriverById(id: Long): Driver = getByIdOrElseThrow(id)
+    suspend fun getDriverById(id: Long): Driver = withContext(Dispatchers.IO) { getByIdOrElseThrow(id) }
```

Comment:
Usually, using context with IO dispatcher should be on repo layer, but in case of spring - on service

In general - possible, but not fully correct

Reply:
fixed